ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixClusterManager.java

```diff
@@ -374,6 +375,17 @@ public PartitionId getPartitionIdFromStream(InputStream stream) throws IOExcepti
     return partition;
   }
 
+  @Override
+  public PartitionId getPartitionIdByName(String partitionIdStr) {
+    byte[] partitionBytes =
```

Comment:
could we just lookup `partitionNameToAmbryPartition.get(partitionIdStr)` here, or is there some inconsistency between the 2 data structures that I'm forgetting about?

Reply:
changed to `partitionNameToAmbryPartition`.